AcMeta/acmeta.adoc

```diff
@@ -1,5 +1,42 @@
 = A metadata convention for processed acoustic data from active acoustic systems
```

Comment:
The table of contents and title are generated automatically when asciidoctor generates the pdf or html files, so it is not necessary add them manually.